s3/pkg/datastore/yig/crypto/client.go

```diff
@@ -0,0 +1,26 @@
+package crypto
```

Comment:
please move this out of the YIG code...we are tracking a separate requirement for Data Encrytion at rest (for object storage), specifically targeting
- Server side encryption SSE
- Client supplied key encryption
- using a KMS

please refer to this and reach out to me, thanks
[https://docs.google.com/document/d/1Fb1bToj_tsWZ0VkYMDMXTR_E6XtWNEIdLw2s4H79C1c/edit?usp=sharing]

Reply:
ok, thanks for your suggestion, i will check it.